R/get-path-from-object.R

```diff
@@ -158,32 +158,188 @@ get_model_id.bbi_model <- function(.mod) {
 
 #' Get path to data file
 #'
-#' Helper to extract the path to the input data from a
-#' `bbi_{.model_type}_summary` or `bbi_{.model_type}_model` object. Note that
-#' **this function only works on models that have been successfully run**
-#' because the path to the data is constructed during model submission.
-#' @param .mod Model to use, either a `bbi_{.model_type}_model` or
-#' `bbi_{.model_type}_summary` object.
+#' Helper to extract the path to the input data from a `NONMEM` control stream file.
```

Comment:
Given this is a generic, this seems too implementation specific for the first sentence of the description. I think the previous description was more appropriate. You could then follow up with more specifics about NONMEM models.